pkg/component/labels/labels.go

```diff
@@ -5,30 +5,43 @@ import (
 	"github.com/redhat-developer/odo/pkg/util"
 )
 
-// ComponentLabel is a label key used to identify the component name
-const ComponentLabel = "app.kubernetes.io/instance"
+// ComponentKubernetesInstanceLabel is a label key used to identify the component name
+const ComponentKubernetesInstanceLabel = "app.kubernetes.io/instance"
 
-// ComponentTypeLabel is Kubernetes label that identifies the type of a component being used
-const ComponentTypeLabel = "app.kubernetes.io/name"
+// ComponentKubernetesNameLabel is Kubernetes label that identifies the type of a component being used
+const ComponentKubernetesNameLabel = "app.kubernetes.io/name"
 
-const ComponentTypeAnnotation = "odo.dev/project-type"
+// ComponentUnknownLabel is the label that is used to display something we do not know
+const ComponentUnknownLabel = "<unknown>"
 
-// ComponentDeployLabel ...
-const ComponentDeployLabel = "Deploy"
+// KubernetesManagedByLabel ...
```

Comment:
Could you keep the same form for the different constant names, either by renaming this one to `ComponentKubernetesManagedByLabel` or by renaming `ComponentKubernetesInstanceLabel` to `KubernetesInstanceLabel`